Container Security/eks-implementation-guide.md

```diff
@@ -22,14 +22,21 @@ The Falcon Container sensor for Linux extends runtime security to container work
 
 ## Pre-requisites
 
+- Existing AWS Account and VPC
+- You will need a workstation with a linux platform
+- Docker installed and running locally on the workstation
```

Comment:
Specifically docker (docker.com) or a Linux container engine?

Reply:
specifically docker is used in these instructions.